upm-tools/AssemblyDefinitions/Bugsnag.asmdef

```diff
@@ -1,3 +1,3 @@
 ﻿{
-	"name": "Bugsnag"
```

Comment:
This text will never be seen by a user and the actual filename of the assembly is not changing, so please revert this change and leave as it was